profiles/common/modules/custom/workbench_moderation_cron/workbench_moderation_cron.module

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/**
+ * @file
+ * Hook implementations of the Workbench Moderation Cron module.
+ */
+
+/**
+ * Implements hook_cron() for workbench_moderation_cron_cron().
+ */
+function workbench_moderation_cron_cron() {
+  global $user;
+
+  $role = user_role_load_by_name("editor");
+  $user->roles = $user->roles + array($role->rid => $role->name);
```

Comment:
Add a role to the global user seem to be risked... (if the user object is saved on other hook_cron, the role will be keept)